runtime/plugins/org.jboss.tools.runtime.core/src/org/jboss/tools/runtime/core/internal/RuntimeCorePreferences.java

```diff
@@ -34,31 +36,54 @@ public static RuntimeCorePreferences getDefault() {
 		String enabledDetectors = getPreferences().get(ENABLED_DETECTORS,null);
 		return enabledDetectors == null ? null : enabledDetectors.split(","); //$NON-NLS-1$
 	}
+	public String[] getDisabledRuntimeDetectors() {
+		String enabledDetectors = getPreferences().get(DISABLED_DETECTORS,null);
```

Comment:
Very minor thing. It doesn't change anything but for code cleanliness sake: "enabledDetectors" -> "disabledDetectors".